cmd/internal/cli/remote.go

```diff
@@ -163,6 +164,16 @@ var remoteAddInsecureFlag = cmdline.Flag{
 	Usage:        "allow connection to an insecure http remote",
 }
 
+// -i|--insecure
+var remoteAddNotDefaultFlag = cmdline.Flag{
+	ID:           "remoteAddNotDefaultFlag",
+	Value:        &remoteAddNotDefault,
+	DefaultValue: false,
+	Name:         "not-default",
```

Comment:
I'm leaning toward `no-default`, even if it's not so gramatically correct, because we have a lot of flags with the general pattern `--no-XXXXX`.

If you feel strongly about it being `not-default` then it's probably a nice idea to have `no-default` working also.